packages/eslint-plugin/package.json

```diff
@@ -5,7 +5,7 @@
   "version": "0.0.0",
   "main": "dist/index.js",
   "scripts": {
-    "build": "swc ./src -d ./dist",
+    "build": "swc ./src -d ./dist --strip-leading-paths --delete-dir-on-start",
```

Comment:
This is ultimately the necessary fix, but the changes to `postinstall` and `preeslint` in the root package.json make things consistent with `pnpm` docs.